cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -329,6 +330,136 @@ def test_course_outline_with_display_course_number_as_none(self):
 
 
 @ddt.ddt
+class TestCourseIndexArchived(CourseTestCase):
```

Comment:
However, these unit tests won't work with #15390 because the rendering is all done Client-side (at least the tests of which course titles are shown in each tab). I think it will be necessary to go to bok choy tests for coverage of which course titles are actually displayed. See https://github.com/edx/edx-platform/pull/15390/files#diff-fcaca8f6c5850d66967cbe21eb1d6348L54.